src/components/nav/Navbar.jsx

```diff
@@ -0,0 +1,36 @@
+import { Outlet, Link } from "react-router-dom";
+
+import "./Navbar.css";
+
+const Navbar = () => {
+  return (
+    <>
+      <Link to="">
+        <img src="../../images/logos/logo copy.png" alt="" />
+      </Link>
+      <nav className="nav">
+        <ul className="nav--ul">
+          <li className="nav--list">
+            <Link to="/products">products</Link>
+          </li>
+          <li className="nav--list">
+            <Link to="/client-resources">client resources</Link>
+          </li>
+          <li className="nav--list">
+            <Link to="/risk-management">risk management</Link>
+          </li>
+          <li className="nav--list">
+            <Link to="/contact">contact</Link>
+          </li>
+          <li className="nav--list">
+            <Link to="/about">about</Link>
+          </li>
+        </ul>

```

Comment:
cand ne am vazut data trecuta am vb despre cum lista de elemente tre sa vina dintr-un `array` de JS si sa avem si `class=selected` pe meniu in functie de pagina pe care esti. urmeaza sa faci asta intr-un PR separat?

Reply:
nu am cum sa fac asta intr-un pr pt ca am facut totul pe branchul `nav`.. fac commit